modules/ROOT/pages/functions/aggregating.adoc

```diff
@@ -16,25 +16,25 @@ For example, given the following query containing two return expressions, `n` an
 RETURN n, count(*)
 ----
 
-The first, `n` is not an aggregate function, so it will be the grouping key.
-The latter, `+count(*)+` is an aggregate expression.
+The first, `n` is not an aggregating function, so it will be the grouping key.
+The latter, `count(*)` is an aggregating function.
 The matching paths will be divided into different buckets, depending on the grouping key.
-The aggregate function will then be run on these buckets, calculating an aggregate value per bucket.
+The aggregating function will then be run on these buckets, calculating an aggregate value per bucket.
 
-The input expression of an aggregation function can contain any expression, including expressions that are not grouping keys.
-However, not all expressions can be composed with aggregation functions.
-The example below will throw an error since `n.x`, which is not a grouping key, is combined with the aggregation expression `+count(*)+`.
+The input expression of an aggregating function can contain any expression, including expressions that are not grouping keys.
+However, not all expressions can be composed with aggregating functions.
+The example below will throw an error since `n.x`, which is not a grouping key, is combined with the aggregating function `count(*)`.
 For more information, see xref:functions/aggregating.adoc#grouping-keys[Grouping keys].
 
 [source, cypher, role=test-skip]
 ----
 RETURN n.x + count(*)
 ----
 
-To use aggregations to sort the result set, the aggregation must be included in the `RETURN` to be used in the `ORDER BY`.
+To sort the result set using aggregating functions, the aggregation must be included in the `RETURN` clause used in the `ORDER BY` clause.
```

Comment:
To me, this sounds like RETURN is part of ORDER BY rather than the other way around. Maybe it can be further re-formulated?